src/aurora/ltrfile.cpp

```diff
@@ -0,0 +1,117 @@
+//
+// Created by patrick on 02.05.18.
+//
+
+#include "src/common/readstream.h"
+#include "src/common/strutil.h"
+
+#include "src/aurora/ltrfile.h"
+#include "src/aurora/resman.h"
+
+static const uint32 kLTRID  = MKTAG('L', 'T', 'R', ' ');
+static const uint32 kVersion10 = MKTAG('V', '1', '.', '0');
+
+static const char* kLetters = "abcdefghijklmnopqrstuvwxyz'-";
```

Comment:
`static const char *kLetters`

Also, I wonder if this works correctly with the Russian version of the game, and also the CJK ones. But that's something for the future. Add a TODO comment about checking this with non-latin games, above the letters, please.